static/assets/migration-guides.svg

```diff
@@ -0,0 +1,69 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<!-- Created with Inkscape (http://www.inkscape.org/) -->
+
+<svg
+   width="600"
+   height="200"
+   viewBox="0 0 600 200"
+   version="1.1"
+   id="svg5"
+   inkscape:version="1.2.2 (b0a8486, 2022-12-01)"
+   sodipodi:docname="migration-guides.svg"
+   xmlns:inkscape="http://www.inkscape.org/namespaces/inkscape"
+   xmlns:sodipodi="http://sodipodi.sourceforge.net/DTD/sodipodi-0.dtd"
+   xmlns="http://www.w3.org/2000/svg"
+   xmlns:svg="http://www.w3.org/2000/svg">
+  <sodipodi:namedview
+     id="namedview7"
+     pagecolor="#000000"
+     bordercolor="#000000"
+     borderopacity="0.25"
+     inkscape:showpageshadow="2"
+     inkscape:pageopacity="0"
+     inkscape:pagecheckerboard="0"
+     inkscape:deskcolor="#d1d1d1"
+     inkscape:document-units="px"
+     showgrid="false"
+     inkscape:zoom="0.4609375"
+     inkscape:cx="421.9661"
+     inkscape:cy="113.89831"
+     inkscape:window-width="1680"
+     inkscape:window-height="919"
+     inkscape:window-x="0"
+     inkscape:window-y="25"
+     inkscape:window-maximized="0"
+     inkscape:current-layer="layer1" />
+  <defs
+     id="defs2">
+    <rect
+       x="15.186441"
+       y="15.186441"
+       width="533.69489"
+       height="143.18643"
+       id="rect113" />
+  </defs>
+  <g
+     inkscape:label="Layer 1"
+     inkscape:groupmode="layer"
+     id="layer1"
+     transform="translate(-32.543398,-212.507)">
+    <text
+       xml:space="preserve"
+       id="text111"
+       style="font-style:normal;font-variant:normal;font-weight:normal;font-stretch:normal;font-size:122px;font-family:'Fira Sans ExtraBold';-inkscape-font-specification:'Fira Sans ExtraBold, ';white-space:pre;shape-inside:url(#rect113);display:inline;fill:#999999"
```

Comment:
I'm fine with the kerning of the text not being perfect for this PR, it's an easy first issue for someone else and should not block this.

That said, this text *should* be exported as a path in inkscape for this PR. At the moment it requires that the end-user has the (non-standard) font available to display correctly. I don't have this font on the system I checked the PR with, so it looks like this instead: 
![image](https://user-images.githubusercontent.com/17050131/224672794-aaac0526-c9b2-4695-9022-c2238614d07b.png)

Reply:
I would see how Fira Mono looks as well, to sidestep the kerning issue, and because we use the monospaced version in other "svgs of code" on the site.

If it gets too wide, maybe a diff-like format could work too?

```
- 0.X
+ 0.Y
```

I like the concept and think you should keep rolling with it :)

Reply:
Great call outs! I'll give this another pass. I like the diff format, will give that a shot. Will also see what Mono looks like to get rid of the kerning.